test/simple.js

```diff
@@ -132,6 +158,37 @@ test("it should not set values for custom serializers/deserializer that return S
     t.end()
 })
 
+test("it should not serialize values for optional properties", t => {
+    var schema = {
```

Comment:
I think a test for { y: undefined } would be nice as well?

Reply:
e717291